### PR TITLE
Tool clean-up

### PR DESCRIPTION
**Pull request by Brotenko**

### CODE REVIEW

File: tools/fix_s_filenames.py [hunk 1]
```diff
@@ -1,31 +0,0 @@
...
-                print(f"??? {f_path}")
-                continue
-
-            func_name = line.split(" ")[1].rstrip()
-            parent_path = os.path.split(f_path)[:-1]
-            new_f_path = os.path.join(*parent_path, func_name + ".s")
-
-            if f_path != new_f_path:
-                print(f"{f_path} => {new_f_path}")
-                os.rename(f_path, new_f_path)
```
Comment (nanaian) [inline tools/fix_s_filenames.py:31]: I'd prefer to keep this

File: tools/mv_segment.py [hunk 2]
```diff
@@ -1,37 +0,0 @@
...
-    print("moving asm/nonmatchings files")
-    os.rename(f"ver/current/asm/nonmatchings/{args.src}", f"ver/current/asm/nonmatchings/{args.dest}")
-
-if os.path.exists(f"ver/current/asm/data/{args.src}.data.s"):
-    print("moving data file")
-    os.rename(f"ver/current/asm/data/{args.src}.data.s", f"ver/current/asm/data/{args.dest}.data.s")
-
-if os.path.exists(f"ver/current/asm/data/{args.src}.rodata.s"):
-    print("moving rodata file")
-    os.rename(f"ver/current/asm/data/{args.src}.rodata.s", f"ver/current/asm/data/{args.dest}.rodata.s")
```
Comment (nanaian) [inline tools/mv_segment.py:37]: I'd prefer to keep this

**Review by nanaian — CHANGES_REQUESTED** [review]
I personally don't see the point of removing tools that work fine..

File: tools/fix_s_filenames.py (hunk 1 above)
Comment (Brotenko) [reply tools/fix_s_filenames.py:31]: Fine by me if you still use/need it!

File: tools/mv_segment.py (hunk 2 above)
Comment (Brotenko) [reply tools/mv_segment.py:37]: Fine by me if you still use/need it!

**Pull request closed without merging**